### DISCUSSION TITLE
Error while installing web3

### DISCUSSION THREAD

**Discussion by Biswa-bob** [smartcontractkit/full-blockchain-solidity-course-py#1533] [General]
Getting an error while installing web3
`ERROR: eth-rlp 0.3.0 has requirement eth-utils<3,>=2.0.0, but you'll have eth-utils 1.10.0 which is incompatible.`

**Comment by cromewar** [comment]
Hello @Biswa-bob 
We will need more info on this, please give a look to #980, and reformulate your question.
Anyway please let us know the following:

1. Your OS
2. Python version
